SpywareFilter/sections/mobile_whitelist.txt

```diff
@@ -0,0 +1,69 @@
+!
+! Exception rules that fix broken mobile apps and mobile websites
+!
+!
+! kiss-anime.ws - broken pages iOS
+!+ PLATFORM(ios)
+@@||kiss-anime.ws/Anime-
+!+ PLATFORM(ios)
+@@||kiss-anime.ws/dl/animeget.php
+!+ PLATFORM(ios)
+@@||kiss-anime.ws/download.php^
```

Comment:
it seems wrong filter.
@BlazDT which filter breaks that site?

Reply:
I guess this one:
`||kiss-anime.ws^$subdocument,~third-party`